docs/_config.yml

```diff
@@ -29,6 +29,7 @@ sidebar_categories:
     - links/batch-http
     - links/dedup
     - links/schema
+    - links/state
```

Comment:
In the TOC the name doesn't match the others:

![image](https://user-images.githubusercontent.com/16724/34170339-5d9fc17e-e49f-11e7-9aed-b0ee65e37649.png)

Reply:
Good call, I just merged @stubailo's PR that fixes this issue :)

Reply:
This is fixed in the deploy preview now 👍